arch/arm/boot/dts/overlays/vc4-kms-v3d-pi4-overlay.dts

```diff
@@ -67,9 +67,11 @@
 	status = "okay";
```

Comment:
The Subject of this commit message is misleading - I was misled - it should be vc4-kms-pi4-v3d.

Reply:
You want the overlay renamed, or the commit message adjusted?
Sorry, I'm not seeing where you've been misled. The Pi4 version of vc4-kms-v3d is surely vc4-kms-v3d-pi4, and it is.

The commit text does want to be reworded as it enables more than it declares as Maxime has made further bits work (eg HDMI) since I created the overlay.

Reply:
The Subject is `overlays: Fix up vc4-kms-pi4 overlay to not enable the VEC.`, which is clearly wrong - the overlay is called `vc4-kms-v3d-pi4`.

Reply:
Your comment is showing up on Github against "[DOWNSTREAM] dtoverlays: Add Pi4 version of vc4-kms-v3d ", hence the confusion.
I'll squash the two together anyway.

Reply:
@pelwell the world would be simpler if `dtoverlay=vc4-kms-v3d` worked on pi3 and pi4 (as `dtoverlay=vc4-fkms-v3d` does). Can you imagine any DT magic that would make a single overlay support both? (firmware side adjustment clearly possible but there may be a nicer option).

Reply:
We had this conversation last week as to whether overlays could have compatible strings that match. The simple answer is not at present.
A firmware side overlay name amendment was what was proposed at least in the medium term.

Reply:
@popcornmix I can't think of a way to make it work within an overlay that didn't require a lot of work figuring out which symbols are actually needed before trying to resolve them.

What would be simpler and safer is an overlay name indirection system - using a text file (e..g. `vc4-kms-v3d.ovl`) containing a mapping from platforms to `.dtbo` files; an absent `.ovl` file would cause it fall back to loading `vc4-kms-v3d.dtbo`.